sphinx_gallery/interactive_example.py

```diff
@@ -332,6 +334,25 @@ def create_jupyterlite_contents(app, exception):
                         os.path.join(contents_dir, i_folder),
                         ignore=_remove_ipynb_files)
 
+    notebook_modification_function = gallery_conf['jupyterlite'][
+        'notebook_modification_function']
+
+    if notebook_modification_function is None:
+        return
+
+    notebook_pattern = os.path.join(contents_dir, "**", "*.ipynb")
+    notebook_filename_list = glob.glob(notebook_pattern, recursive=True)
```

Comment:
`glob` order is arbitrary, seems like it would be nice to sort for consistency
```suggestion
    notebook_filename_list = sorted(
        glob.glob(notebook_pattern, recursive=True))
```

Reply:
Done